modules/service/src/main/resources/db/migration/V1016__cfp_title_override.sql

```diff
@@ -0,0 +1,87 @@
+-- Add a nullable title override column.
+ALTER TABLE t_cfp
+  ADD COLUMN c_title_override text NULL DEFAULT NULL CHECK (c_title_override IS NULL OR length(c_title_override) > 0);
+
+-- Drop the v_cfp view which calls the title formatting function.
+DROP VIEW v_cfp;
+
+-- Update the title formatting function to consider the title override.
+DROP FUNCTION format_cfp_title;
+CREATE FUNCTION format_cfp_title(
+  cfpType       e_cfp_type,
+  semester      d_semester,
+  titleOverride text,
+  startDate     date,
+  endDate       date,
+  instruments   d_tag[]
+) RETURNS text AS $$
+DECLARE
+  instrument_list text;
+  name text;
+BEGIN
+  SELECT c_name INTO name FROM t_cfp_type WHERE c_type = cfpType;
+  instrument_list := array_to_string(instruments, ', ');
+  IF (titleOverride IS NOT NULL) THEN
+    RETURN titleOverride;
+  ELSE
```

Comment:
This function is the same as before, except for the addition of the `titleOverride` parameter and this `IF` that respects it.